dspace/modules/dryad-widgets/dryad-widgets-webapp/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Does this conflict with the log4j.properties in `dspace/config`? even if not, it can be confusing, and there should only be one log4j.properties in the repo